diagnostic.md

```diff
@@ -3,36 +3,42 @@
 What are three advantages of using Ember?
 
 ```sh
-# your answer here
+- Each view state has its own URL - managed by routes
+- Writing less AJAX calls.
+- data display in the front end is faster due to fewer ajax calls.
 ```
 
 What is the templating system used in Ember? How it is related to
 handlebars?
 
 ```sh
-# your answer here
+components.
```

Comment:
It is Handlebars! It also has some extra helpers in it, and it can reference components
by name.